pymc/model_graph.py

```diff
@@ -102,11 +102,42 @@ def get_parents(self, var: TensorVariable) -> Set[VarName]:
         parents = self._get_ancestors(var, func)
         return self._filter_parents(var, parents)
 
-    def make_compute_graph(self) -> Dict[str, Set[VarName]]:
+    def vars_to_plot(self, selected_vars=None):
+        if selected_vars is None:
+            return self.var_names
+        else:
+            selected_vars = set(selected_vars)
```

Comment:
because of the return in the line above you can do
```suggestion
        selected_vars = set(selected_vars)
```
Ideally give it a new name, because it has a different type now: `sel_names: Set[str]`